src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/CourseController.java

```diff
@@ -55,17 +57,24 @@ public String index(Model model, OAuth2AuthenticationToken token) throws AccessF
             logger.info("scheduleItemRepository="+scheduleItemRepository);
             Iterable<ScheduleItem> myclasses = scheduleItemRepository.findByUid(uid);
             // logger.info("there are " + myclasses.size() + " courses that match uid: " + uid);
+            String[] days = new String[]{"Monday","Tuesday","Wednesday","Thursday","Friday"};
```

Comment:
Any reason why "Saturday" and "Sunday" aren't on here? I know that we usually don't have classes on those days, but it's technically possible.

Reply:
Since GOLD didn't have Saturday and Sunday, we decided to follow their convention

Reply:
Fair point.